File: Data Structures & Algorithms/majority-element/submission-1.go
func majorityElement(nums []int) int {
    cnt := 0 
    candidate := 0

    for _, num := range nums{
        if cnt == 0{
            candidate = num
        }

        if candidate == num {
            cnt++
        }else {
            cnt--
        }
    }
    return candidate
}

File: Data Structures & Algorithms/length-of-last-word/submission-1.py
class Solution:
    def lengthOfLastWord(self, s: str) -> int:
        cnt = 0

        for i in range(len(s)-1,-1,-1):
            if s[i] != " ":
                cnt +=1
            elif cnt >0 and s[i] == " ":
                break
        
        return cnt 